packages/core/test/Form_test.js

```diff
@@ -2226,6 +2226,33 @@ describeRepeated("Form common", createFormComponent => {
     });
   });
 
+  describe("Form readonly prop", () => {
+    const schema = {
+      type: "object",
+      properties: {
+        foo: { type: "string" },
+        bar: { type: "string" },
+      },
+    };
+    const formData = { foo: "foo", bar: "bar" };
+
+    it("should not have any readonly items", () => {
+      const { node } = createFormComponent({ schema, formData });
+
+      expect(node.querySelectorAll("input:read-only")).to.have.length.of(0);
+    });
+
+    it("should readonly all items", () => {
```

Comment:
Can we add a test just to ensure it works on nested objects / arrays?

Reply:
Sure... I originally followed the pattern for the `disabled` tests above. Should those also be updated?

Reply:
Updated the one test to use a nested object

Reply:
<img width="650" alt="Screen Shot 2021-09-24 at 4 53 46 PM" src="https://user-images.githubusercontent.com/51679588/134750346-54da2042-140a-413a-adcc-b9e9ec6af0ea.png">

The playground works with arrays